src/utils/hooks.js

```diff
@@ -3,15 +3,23 @@ import { useLocation, useParams } from 'react-router-dom';
 export function useEmbed() {
   // Check if we're embedded in an iFrame
   const { pathname } = useLocation();
-  const { id } = useParams();
+  const { id, countyId } = useParams();
   const isEmbed = pathname.includes('/embed');
   const protocol = window.location.protocol;
   const hostname = window.location.hostname;
+
+  let path = `us/`;
+  if (id) {
```

Comment:
this is not handling the countyFipsId path

Reply:
ahh i see this is just for copying the embed url and we don't need to support both in this case

Reply:
Correct.